### PR TITLE
Update kubernetes_orchestration example

### PR DESCRIPTION
**Pull request by fa9r**
## Describe changes
I updated the `kubernetes_orchestration` example according to the Kubernetes orchestration blog post.

## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/features/integrations) table.
- [ ] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Other (add details above)

### CODE REVIEW

**Review by AlexejPenner — COMMENTED** [review]
Edited as I saw its already in the form of the best practices.

File: examples/kubernetes_orchestration/README.md [hunk 1]
```diff
@@ -126,10 +139,25 @@ zenml stack register k8s_stack
...
+```bash
+zenml stack up
+```
+
+If everything went well, you should see logs messages similar to the following
+in your terminal:
+
+![zenml stack up output](assets/zenml_stack_up_output.png)
 
 ## :computer: Run Pipeline Locally
```
Comment (AlexejPenner) [inline examples/kubernetes_orchestration/README.md:162]: This heading might be a bit confusing to the reader? Didn't we just set up a Kubernetes cluster in the cloud, why is this running locally.

Comment (fa9r) [reply examples/kubernetes_orchestration/README.md:162]: Good point, I added a bit more explanation what's going on there.

Comment (htahir1) [reply examples/kubernetes_orchestration/README.md:164]: I would even change the heading title to "Run the pipeline"

**Review by htahir1 — COMMENTED** [review]
Left one comment :-)

File: examples/kubernetes_orchestration/README.md [hunk 2]
```diff
@@ -184,21 +196,36 @@ E.g., you can use these labels to get a list of all pods related to the
...
 zenml stack down --force
 ```
+
+### Delete Infrastructure Resources
+Lastly, if needed, spin down the infrastructure you created.
+How to do this depends on which cloud provider you were using to run this
+example.
+
+If you have used our Terraform script to spin up the resources, simply run
+```
```
Comment (strickvl) [inline examples/kubernetes_orchestration/README.md:229]:
```suggestion
If you have used our Terraform script to spin up the resources, simply run:

```shell
```

File: .pyspelling-ignore-words [hunk 3]
```diff
@@ -724,6 +725,7 @@ mkdir
 mkdocs
 mlflow
 mlmd
+mlops
```
Comment (htahir1) [inline .pyspelling-ignore-words:728]: Good word to ignore :-D But I guess we go with MLOps as the way to write it?

Comment (strickvl) [reply .pyspelling-ignore-words:728]: The problem is that we're referencing the repository, and I haven't figured out how to get the PySpelling filter to work to ignore URLs, so I think for now at least we have to add it.

File: examples/kubernetes_orchestration/README.md (hunk 1 above)
Comment (fa9r) [reply examples/kubernetes_orchestration/README.md:164]: Ah, yeah makes sense!

File: examples/kubernetes_orchestration/README.md (hunk 2 above)
Comment (fa9r) [reply examples/kubernetes_orchestration/README.md:229]: Done.

**Review by strickvl — APPROVED** [review]

**Review by htahir1 — APPROVED** [review]

**Pull request merged**